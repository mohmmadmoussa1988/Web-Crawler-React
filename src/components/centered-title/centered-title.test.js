import React from 'react';
import {configure,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CenteredTitle from './centered-title';
import toJson from 'enzyme-to-json';
configure({adapter:new Adapter()});
describe(`<CenteredTitle />`,()=>{

    it('should render CenteredTitle',()=>{
        const wrapper = shallow(<CenteredTitle/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })


})