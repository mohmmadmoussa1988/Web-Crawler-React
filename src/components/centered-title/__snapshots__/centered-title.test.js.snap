// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CenteredTitle /> should render CenteredTitle 1`] = `<styled.h2 />`;
